### PR TITLE
Siafund

### PR DESCRIPTION
**Pull request by Mingling94**
On top of #140 and some of #141 

Can view transactions, siacoin or siafund. Can also make siafund transactions now.

TODO after this PR:
- Add wallet settings page with the following:
  - Filter what transactions one views
  - 'Siafund' mode switch that unlocks this so one can't accidentally send siafunds. Perhaps even have another mode switch in the general UI so one REALLY has to be sure of sending siafunds.
  - Load Legacy Wallet button

### CODE REVIEW

**Comment by DavidVorick** [comment]
hold off on the settings page for now, it's more important to have a file browser that can do folders.

**Comment by lukechampine** [comment]
needs rebase

**Comment by Mingling94** [comment]
Boop @lukechampine @DavidVorick is the code in this PR good to go?

I'm doing a bunch of fixes on top of this but I figure there's too many commits in this PR already. I'll make another PR for it.

**Comment by lukechampine** [comment]
it needs to be rebased

**Comment by Mingling94** [comment]
Ah, sorry. Github said it could be merged automatically, so I didn't think it needed rebasing. Do you prefer not to trust automatic (conflict-less) rebasing? @lukechampine 

Regardless, it's rebased now.

**Comment by lukechampine** [comment]
excessive merging leads to a messy commit history. Rebasing keeps the history linear and avoids duplicate commits. Whether you rebase or merge, you wind up with the same code, but I find rebasing to be better practice for PRs.
And yes, there are situations where git's auto-merge algorithm can have unexpected results. I've seen it report no conflicts after merging files in completely insane ways. When you rebase, you have to deal with those conflicts manually, which can be a pain, but is safer in the long run.

**Comment by Mingling94** [comment]
Word, well it's rebased now :P

File: plugins/Wallet/js/sending.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+	var address = $('#transaction-address').val();
+
+	// Verify number
+	if (!isNumber(value)) {
+		tooltip('Enter numeric value to send!', caller);
+		return;
+	} 
+	// Verify balance
+	var total = new BigNumber(value).times(unit);
+	// TODO: Sending siafunds is gravitous. Should make the whole wallet have a
```
Comment (lukechampine) [inline plugins/Wallet/js/sending.js]: I get what you mean, but that's...not a word. lol

Comment (Mingling94) [inline plugins/Wallet/js/sending.js]: I needz spall check for my vem >3<

**Pull request merged**